wire-ios/Wire-iOS/Sources/UserInterface/StartUI/StartUI/StartUIViewController+NavigationController.swift

```diff
@@ -0,0 +1,67 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import UIKit
+
+extension StartUIViewController {
+
+    func setupNavigationBar() {
+        navigationController?.navigationBar.barTintColor = backgroundColor
+        navigationController?.navigationBar.isTranslucent = false
+        navigationController?.navigationBar.tintColor = UIColor.accent()
+        navigationController?.navigationBar.titleTextAttributes = DefaultNavigationBar.titleTextAttributes()
+    }
+
+    func setupNavigationBarButtonItems() {
+        let cancelButton = UIBarButtonItem(
+            title: L10n.Localizable.General.cancel,
+            style: .plain,
+            target: self,
+            action: #selector(onDismiss)
+        )
+
+        cancelButton.accessibilityLabel = L10n.Accessibility.ContactsList.CancelButton.description
+        cancelButton.accessibilityIdentifier = "cancel"
+
+        navigationItem.rightBarButtonItem = cancelButton
+
+        let createGroupButton = UIBarButtonItem(
+            title: L10n.Localizable.Peoplepicker.Button.createConversation,
+            style: .plain,
+            target: self,
+            action: #selector(createGroup)
+        )
```

Comment:
it seems in design the button is bolder -> https://www.figma.com/design/sOAbcd6UVtlZ5ejK4BoEvX/iOS?node-id=349-4768&t=emGf1FY69JMZEO3f-4

Reply:
Fixed it here: https://github.com/wireapp/wire-ios/commit/a70a6b47b3858c753734eb549cb463b297506f6f